src/OpenSage.Game/Graphics/ModelMesh.cs

```diff
@@ -71,9 +72,7 @@ public sealed class ModelMesh : DisposableBase
 
             _effect = effect;
 
-            _vertexBuffer = AddDisposable(graphicsDevice.CreateStaticBuffer(
-                vertices,
-                BufferUsage.VertexBuffer));
+            _vertexBuffer = AddDisposable(graphicsDevice.CreateVertexBuffer(vertexData));
```

Comment:
Indices and vertices should use the same `CreateXXX` function

Reply:
But `CreateStaticBuffer` takes a `T[]`, while `CreateVertexBuffer` takes a `byte[]`. I could name them both `CreateStaticBuffer`, if you'd prefer?